src/components/cards/ImpulseCard.vue

```diff
@@ -38,11 +49,12 @@ export default {
 </script>
 <style lang="scss" scoped>
```

Comment:
I like your new redesign great work. What I would improve is this view problem I have:
![image](https://user-images.githubusercontent.com/29863859/102588174-50264e00-410d-11eb-9304-87cc18d5cb97.png)
If you look to the button, the card frame disappears in our navbar.